### ISSUE TITLE
`activate --revision project-name:latest` feature?

### ISSUE THREAD

**Issue by trianglegrrl** [ember-cli/ember-cli-deploy#121]
Wow, I think I speak for my whole team when I say ember-cli-deploy is AWESOME! We were worried it would take a ton of time, but we have it running with minimal effort.

Just one request for help (or a new feature): is there a simple way to activate the latest revision? Being able to do:

```
ember deploy:activate --revision my-project:latest
```

Would be super boss.

**Comment by achambers** [comment]
Thanks for the feedback @trianglegrrl :+1:

That functionality doesn't currently exist but it is in our radar. Hopefully we can get it in soon for you :)

**Comment by motdotla** [comment]
:+1: would love to see this too.

**Comment by elbuo8** [comment]
:+1:

**Comment by quiddle** [comment]
:+1:

**Comment by blimmer** [comment]
This would be a really lovely feature.

**Comment by avaz** [comment]
+1

**Comment by devinus** [comment]
:+1:

**Comment by epogue** [comment]
:+1:

**Comment by wongpeiyi** [comment]
:+1:

I'm using a super hacky Bash script for that now:
```bash
function eda() {
  list=`ember deploy:list`;
  rev=`echo $list | sed -n "s/.*revisions: | [=>[:space:]]*\([^[:space:]]*\).*/\1/p"`;
  printf "Activating $rev \n";
  ember deploy:activate --revision $rev;
}
```

**Comment by lukemelia** [comment]
FYI, this is definitely going to be part of 0.5.0.

**Comment by mtmckenna** [comment]
Looking forward to this feature! I'm also interested in being able to deploy and activate in one step, which seems related. I've been doing this via shell script, but I think it'd be a nice feature to have built in.

The shell script I've been using:

```
export AWS_SECRET_KEY=`cat .aws_secret_key`
ENVIRONMENT=production

ember deploy --environment=$ENVIRONMENT

if [ $? -eq 0 ]
then
  LATEST_REVISION="$(ember deploy:list --environment=$ENVIRONMENT | grep "1)" | awk '{print $2}')"
  ember deploy:activate --revision=$LATEST_REVISION --environment=$ENVIRONMENT
  echo "$(tput setaf 2)Deployed the heck out of the latest revision."
  exit 0
else
  echo "$(tput setaf 1)Something is jacked up with the deploy."
  exit 1
fi
```

I put the script in this gist in case it's handy: https://gist.github.com/mtmckenna/2bbce8c14f520c78088b

**Comment by achambers** [comment]
Matt, 0.5.0 allows you to do this with `ember deploy production --activate`

**Comment by mtmckenna** [comment]
Awesome! Thank you @achambers !

**Comment by lukemelia** [comment]
Addressed by 0.5.0, which is now released!
